include/comments.php

```diff
@@ -3,12 +3,37 @@
 function createComment($content){
     dbQuery(
         '
-        INSERT INTO comments (content)
-        VALUES(:content)
+        INSERT INTO comments (content, blogPostId)
+        VALUES(:content, :blogPostId)
         ',
         [
-            'content'=>$_REQUEST['content']
+            'content'=>$content,
+            'blogPostId'=>$_REQUEST['blogPostId']
         ]
     );
-return $content;
-}
\ No newline at end of file
+    return $content;
+}
+
+function getAllComments($blogPostId){
```

Comment:
This name probably isn't appropriate not that it's just getting comments for a specific blog post